azure/packages/azure-local-service/package.json

```diff
@@ -34,9 +34,9 @@
 		"@biomejs/biome": "~1.9.3",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.58.3",
-		"@fluidframework/eslint-config-fluid": "^6.0.0",
-		"eslint": "~8.55.0",
-		"eslint-config-prettier": "~9.0.0",
+		"@fluidframework/eslint-config-fluid": "~6.1.0",
```

Comment:
Unlike the other eslint packages I think our own one should stay with `^` deps as its (as of recently I think) trying to follow semver with respect to our needs.

Reply:
Bumping this to ^6.1.0 would be fine though